hi-ml-multimodal/README.md

```diff
@@ -89,7 +114,7 @@ If you use our code or models in your research, please cite [our paper][7] (pres
 [4]: https://mybinder.org/v2/gh/microsoft/hi-ml/HEAD?labpath=hi-ml-multimodal%2Fnotebooks%2Fphrase_grounding.ipynb
 [5]: https://docs.conda.io/en/latest/miniconda.html
 [6]: https://aka.ms/biovil-models
-[7]: https://link.springer.com/chapter/10.1007/978-3-031-20059-5_1
-[8]: https://eccv2022.ecva.net/
+[7]: https://doi.org/10.1007/978-3-031-20059-5_1
+[8]: https://openreview.net/forum?id=5jScn5xsbo
```

Comment:
This link is private -- Please use the Arxiv link instead.

Reply:
Oops. Is it sensitive? Shall I revert and force-push?